flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/security_scan/.trivyignore

```diff
@@ -1 +0,0 @@
-CVE-2021-33430
```

Comment:
does it really makes sense to remove it?

Reply:
Hm, my thought was actually to keep the flavor as simple as possible.

Reply:
I added the .trivyignore again and also added a comment why this file is empty